tools/content-validation/eng/pipelines/python/python-sdk-docs-content-validation.yml

```diff
@@ -0,0 +1,229 @@
+  jobs:
+  - job: fetch
+    displayName: "Getting all packages to be tested"
+    pool:
+      vmImage: ubuntu-latest
+    steps:
+    - task: UseDotNet@2
+      displayName: 'Install .NET Core SDK'
+      inputs:
+        packageType: 'sdk'
+        version: '8.0.x'
+        installationPath: $(Agent.ToolsDirectory)/dotnet 
+
+    - task: DotNetCoreCLI@2
+      displayName: New Manifest for tool
+      inputs:
+        command: custom
+        custom: 'new '
+        arguments: tool-manifest
+
+    - task: DotNetCoreCLI@2
+      displayName: Install Playwright CLI
+      inputs:
+        command: custom
+        custom: 'tool '
+        arguments: install Microsoft.Playwright.CLI
+
+    - task: DotNetCoreCLI@2
+      displayName: Build all projects and libraries
+      inputs:
+        command: 'build'
+        projects: 'content-validation.sln'
+
+    - task: DotNetCoreCLI@2
+      displayName: Run Playwright Install
+      inputs:
+        command: custom
+        custom: 'tool '
+        arguments: run playwright install
+
+    - task: DotNetCoreCLI@2
+      displayName: Running project PendingTestingPackagesThisMonth
+      env:
+        Language: 'Python'
+      inputs:
+        command: 'run'
+        workingDirectory: '$(System.DefaultWorkingDirectory)/PendingTestingPackagesThisMonth'
+      name: runPackagesJob
+
+    - pwsh: |
+        $matrixJson = Get-Content "$(System.DefaultWorkingDirectory)/eng/pipelines/packages.json" -Raw
+        Write-Host "##vso[task.setvariable variable=matrixJson;isOutput=true]$matrixJson"
+        Write-Host "DEBUG: matrixJson = $matrixJson"
+      name: setMatrixVar
+
+  - job: triggerSelf
+    displayName: Trigger self pipeline
+    dependsOn: fetch
+    pool:
+      vmImage: ubuntu-latest
+    variables:
+      MATRIX_JSON: $[ dependencies.fetch.outputs['setMatrixVar.matrixJson'] ]
+      MAX_PARALLEL: ${{ parameters.maxParallel }}
+
+    steps:
+    - checkout: self
+      persistCredentials: true
+
+    - task: PowerShell@2
+      displayName: Trigger self pipeline with package matrix
+      env:
+        SYSTEM_ACCESSTOKEN: $(System.AccessToken)
+        MAX_PARALLEL: $(MATRIX_JSON) 
+        MATRIX_JSON: $(MAX_PARALLEL)
+      inputs:
+        targetType: inline
+        script: |
+          $orgUri     = "$(System.CollectionUri)"
+          $proj       = "$(System.TeamProject)"
+          $pipelineId = $(System.DefinitionId)
+          $body = @{
+            resources = @{
+              repositories = @{
+                self = @{
+                  refName = "$(Build.SourceBranch)"
+                }
+              }
+            }
+            templateParameters = @{
+              packages    = $env:MATRIX_JSON
+              maxParallel = $env:MAX_PARALLEL
+            }
+          } | ConvertTo-Json -Depth 10
+
+          Write-Host ">> Triggering pipeline #$pipelineId in $proj with:"
+          Write-Host $body
+
+          Invoke-RestMethod `
```

Comment:
You shouldn't need to trigger the pipeline again to get the dynamic matrix. You can dynamically create the matrix like you are and then it will define the set of jobs in the same pipeline run.